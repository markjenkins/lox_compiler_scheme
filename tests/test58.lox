if (false)
   print "never happens";
else
   print "false happens";
