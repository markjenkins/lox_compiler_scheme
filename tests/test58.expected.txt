OP_FALSE
OP_JUMP_IF_FALSE 7
OP_POP
OP_CONSTANT "never happens"
OP_PRINT
OP_JUMP 4
OP_POP
OP_CONSTANT "false happens"
OP_PRINT
OP_RETURN
